<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionReveal_One_Peg"/>
    <addaction name="actionResign"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuGameMode">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Game Mode</string>
     </property>
     <property name="icon">
      <iconset theme="emblem-favorite"/>
     </property>
     <property name="separatorsCollapsible">
      <bool>true</bool>
     </property>
     <addaction name="actionGameMode_MVH"/>
     <addaction name="actionGameMode_HVM"/>
     <addaction name="actionGameMode_HVH"/>
    </widget>
    <addaction name="menuGameMode"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Indicators"/>
    <addaction name="actionAuto_Set_Pins"/>
    <addaction name="actionAuto_Close_Rows"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionQtMind_Home_Page"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_QtMind"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="list-add" resource="../resource.qrc">
     <normaloff>:/icons/new-icon.png</normaloff>:/icons/new-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionResign">
   <property name="icon">
    <iconset theme="list-remove" resource="../resource.qrc">
     <normaloff>:/icons/resign.png</normaloff>:/icons/resign.png</iconset>
   </property>
   <property name="text">
    <string>Resign</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close" resource="../resource.qrc">
     <normaloff>:/icons/close-icon.png</normaloff>:/icons/close-icon.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAuto_Set_Pins">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="task-accepted"/>
   </property>
   <property name="text">
    <string>Auto Set Pins</string>
   </property>
  </action>
  <action name="actionAuto_Close_Rows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="task-complete"/>
   </property>
   <property name="text">
    <string>Auto Close Rows</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset theme="configure"/>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout_QtMind">
   <property name="icon">
    <iconset theme="dialog-information" resource="../resource.qrc">
     <normaloff>:/icons/hicolor/64x64/qtmind.png</normaloff>:/icons/hicolor/64x64/qtmind.png</iconset>
   </property>
   <property name="text">
    <string>About QtMind</string>
   </property>
   <property name="toolTip">
    <string>About QtMind</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/qt_icon.png</normaloff>:/icons/qt_icon.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionGameMode_MVH">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="cpu" resource="../resource.qrc">
     <normaloff>:/icons/modemaster.png</normaloff>:/icons/modemaster.png</iconset>
   </property>
   <property name="text">
    <string>Machine vs Human</string>
   </property>
  </action>
  <action name="actionGameMode_HVM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="meeting-chair" resource="../resource.qrc">
     <normaloff>:/icons/modebreaker.png</normaloff>:/icons/modebreaker.png</iconset>
   </property>
   <property name="text">
    <string>Human vs Machine</string>
   </property>
  </action>
  <action name="actionReveal_One_Peg">
   <property name="icon">
    <iconset theme="system-help" resource="../resource.qrc">
     <normaloff>:/icons/reveal_one_peg.png</normaloff>:/icons/reveal_one_peg.png</iconset>
   </property>
   <property name="text">
    <string>Reveal One Peg</string>
   </property>
  </action>
  <action name="actionQtMind_Home_Page">
   <property name="icon">
    <iconset theme="applications-internet" resource="../resource.qrc">
     <normaloff>:/icons/globe-icon.png</normaloff>:/icons/globe-icon.png</iconset>
   </property>
   <property name="text">
    <string>QtMind Home Page</string>
   </property>
   <property name="toolTip">
    <string>Go to QtMind home page</string>
   </property>
  </action>
  <action name="actionShow_Indicators">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="layer-visible-on"/>
   </property>
   <property name="text">
    <string>Show Indicators</string>
   </property>
  </action>
  <action name="actionGameMode_HVH">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="system-users">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Human vs Human</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
